src/pycryptopp/test/test_ed25519_kat.py

```diff
@@ -37,8 +37,8 @@ def test_short(self):
             vk2 = ed25519.VerifyingKey(vk_s)
             self.failUnlessEqual(vk2, vk) # objects should compare equal
             newsig = sk.sign(msg)
-            sig_R,sig_S = sig[:32],sig[32:]
-            newsig_R,newsig_S = newsig[:32],newsig[32:]
+            #sig_R,sig_S = sig[:32],sig[32:]
+            #newsig_R,newsig_S = newsig[:32],newsig[32:]
             self.failUnlessEqual(hexlify(newsig), hexlify(sig)) # deterministic sigs
```

Comment:
I'd say just delete the lines, instead.  Alternatively, rewrite them as English comments.  "The first 32 bytes of sig are ....  The rest of the bytes are ...".